commit.go

```diff
@@ -123,16 +126,18 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
 	}
 
 	img, err := is.Transport.GetStoreImage(b.store, dest)
-	if err == nil {
-		if options.IIDFile != "" {
-			if err := ioutil.WriteFile(options.IIDFile, []byte(img.ID), 0644); err != nil {
-				return errors.Wrapf(err, "failed to write Image ID File %q", options.IIDFile)
-			}
-		} else {
-			fmt.Printf("%s\n", img.ID)
+	if err != nil {
```

Comment:
@mheon, I've been playing/testing this this morning and unless @nalind yells,  you want to add something like:
```
# git diff
diff --git a/commit.go b/commit.go
index 82e57cd..401f790 100644
--- a/commit.go
+++ b/commit.go
@@ -126,6 +126,11 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
        }
 
        img, err := is.Transport.GetStoreImage(b.store, dest)
+
+       if (err == storage.ErrImageUnknown) {
+               return "", nil
+       }
+
        if err != nil {
                return imgID, errors.Wrapf(err, "failed to retrieve ID of committed image")
        }
```
The only thing a little wonky is with this in place is the 'buildah commit` leaves an extra blank line.

```
# git diff
diff --git a/commit.go b/commit.go
index 82e57cd..401f790 100644
--- a/commit.go
+++ b/commit.go
@@ -126,6 +126,11 @@ func (b *Builder) Commit(ctx context.Context, dest types.ImageReference, options
        }
 
        img, err := is.Transport.GetStoreImage(b.store, dest)
+
+       if (err == storage.ErrImageUnknown) {
+               return "", nil
+       }
+
        if err != nil {
                return imgID, errors.Wrapf(err, "failed to retrieve ID of committed image")
        }
```

With the above in place, the tests pass.